homeassistant/components/tessie/strings.json

```
@@ -410,5 +410,40 @@
     "no_cable": {
       "message": "Insert cable to lock"
     }
+  },
+  "issues": {
+    "deprecated_speed_limit_entity": {
+      "title": "Speed limit lock entity deprecated",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "Detected Tessie speed limit lock entity usage",
+            "description": "The Tessie integration's speed limit lock entity has been deprecated and will be remove in 2024.11.0.\nHome Assistant detected that entity `{entity}` is being used in `{info}`\n\nYou should remove the speed limit lock entity from `{info}` then click submit."
```

```suggestion
            "description": "The Tessie integration's speed limit lock entity has been deprecated and will be remove in 2024.11.0.\nHome Assistant detected that entity `{entity}` is being used in `{info}`\n\nYou should remove the speed limit lock entity from `{info}` then click submit to fix this issue."
```
I suppose there is no replacement here so the users are left with nothing once this has been removed?